includes/plugins/wc-memberships/class-memberships.php

```diff
@@ -855,6 +855,11 @@ private static function can_manage_woocommerce( $caps ) {
 	 * @return array Filtered capabilities.
 	 */
 	public static function user_has_cap( $all_caps, $caps, $args ) {
+		global $wp_query;
```

Comment:
I wonder if `did_action('wp') > 0` would be more accurate here? To know if conditional tags are available?

Reply:
Gets the job done in a more elegant way, thanks!